.tekton/jobset-main-pull-request.yaml

```diff
@@ -0,0 +1,629 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift/kubernetes-sigs-jobset?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
```

Comment:
We can only trigger image build for some set of files https://github.com/openshift/kubernetes-sigs-lws/blob/main/.tekton/lws-main-pull-request.yaml#L12-L27. For example, there is no need to trigger an image build when there is a change in Dockerfile.ci.

Reply:
But this can be handled in a followup PR as well.

Reply:
Ack, updated to include the jobset files.